tutorials/IBC-connection.md

```diff
@@ -0,0 +1,309 @@
+git checkout v2.4.2
+make install
+```
+
+Verify your rly version with `rly version`:
+
+```bash
+rly version
+```
+
+It should return:
+
+```bash
+version: 2.4.2
+commit: 259b1278264180a2aefc2085f1b55753849c4815 (dirty)
+cosmos-sdk: v0.47.5
+go: go1.21.4 darwin/arm64
+```
+
+### Setup relayer config
+
+Firstly, generate an empty config file with this command:
+
+```bash
+rly config init
+```
+
+Afterward, edit the file in the directory `$HOME/.relayer/config/config.yaml` with any text editor you prefer. Then, paste this configuration into it.
+
+```yaml
+global:
+    api-listen-addr: :5183
+    timeout: 10s
+    memo: ""
+    light-cache-size: 20
+chains:
+    osmo-local:
+        type: cosmos
+        value:
+            key-directory: /root/.relayer/keys/osmosis-testnet-1
+            key: default
+            chain-id: osmosis-testnet-1
+            rpc-addr: http://localhost:46657 
+            account-prefix: osmo
+            keyring-backend: test
+            gas-adjustment: 1.5
+            gas-prices: 0.1uosmo
+            min-gas-amount: 100000
+            max-gas-amount: 1000000
+            debug: true
+            timeout: 20s
+            block-timeout: ""
+            output-format: json
+            sign-mode: direct
+            extra-codecs: []
+            coin-type: 118
+            signing-algorithm: ""
+            broadcast-mode: batch
+            min-loop-duration: 0s
+            extension-options: []
+            feegrants: null
+    gm-local:
+        type: cosmos
+        value:
+            key-directory: /root/.relayer/keys/test
+            key: default
+            chain-id: gm
+            rpc-addr: http://localhost:36657
+            account-prefix: gm
+            keyring-backend: test
+            gas-adjustment: 1.5
+            gas-prices: 0.1stake
+            min-gas-amount: 100000
+            max-gas-amount: 1000000
+            debug: true
+            timeout: 20s
+            block-timeout: ""
+            output-format: json
+            sign-mode: direct
+            extra-codecs: []
+            coin-type: 118
+            signing-algorithm: ""
+            broadcast-mode: batch
+            min-loop-duration: 0s
+            extension-options: []
+            feegrants: null
+paths: {}
+```
+
+### Create relayer account
+
+Add keys for each chain with this command
+
+The mnemonic-words is the mnemonic that you have when init local node, ensure that each wallet have token to start relaying.
+
+```bash
+rly keys restore osmo-local default "milk verify alley price trust come maple will suit hood clay exotic"
+rly keys restore gm-local default "milk verify alley price trust come maple will suit hood clay exotic"
+```
+
+### Create IBC channel
+
+Create a new blank path to be used in generating a new path (connection and client) between two chains
+
+```bash
+rly paths new osmosis-testnet-1 gm osmo-gm
+```
+
+and then you can create channel with this command
+
+```bash
+rly transact link osmo-gm
```

Comment:
I've gone through a few times and wiped my machine clean, but running into issues with this command:

```bash
$ rly transact link osmo-gm

Error: error creating clients: failed to query latest heights: node at http://localhost:36657/ running chain gm not caught up
```

Reply:
resolved with new rollkit version